src/repository/screens/repository.screen.js

```diff
@@ -99,25 +99,46 @@ class Repository extends Component {
       changeStarStatusRepoByDispatch,
       forkRepoByDispatch,
       navigation,
-      username,
     } = this.props;
-    const showFork = repository.owner.login !== username;
 
     if (index === 0) {
+      // Share
+      this.shareRepository(repository);
+    }
+
+    if (index === 1) {
+      // Star
       changeStarStatusRepoByDispatch(
         repository.owner.login,
         repository.name,
         starred
       );
     }
 
-    if (index === 1 && showFork) {
```

Comment:
Oops nvm, there's already a check to not have it included in line 173, we're good :+1: